index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Njoroge</title>
+</head>
+
+<body>
+    <div class="container">
+        <header class="top-header">
+            <a href="https://github.com/francis450" class="header-item logo">My logo</a>
+            <nav class="header-item"><img src="img/Union.svg" alt=""></nav>
+        </header>
+
+        <section>
+            <h1 class="intro">
+                I'm Francis<br>
+                Glad to see you!
+            </h1>
+
+            <p class="detail">
+                I'm a software developer!
+                I can help you build a product ,
+                feature or website Look through
+                some of my work and experience! If
+                you like what you see and have a project you need
+                coded, don't hestiate to contact me.
+            </p>
```

Comment:
- [ ] Good job creating this page with a beautiful design :clap: , however  I noticed that the font-family is `Italic` while the required on is `normal` font-family, Kindly make sure you update your design :+1: 

|  #Your Design    | #Figma Design    |
|  :---: | :---: |
| ![Screenshot from 2023-01-13 17-10-17](https://user-images.githubusercontent.com/80458199/212369871-bd5a0025-b2aa-41ba-92b1-38a886acca2b.png) |  ![Screenshot from 2023-01-13 18-24-41](https://user-images.githubusercontent.com/80458199/212370018-b60c40fc-8cfb-4641-b869-2a8f4761dc01.png)
     |